docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Set Up Code Builder
+lang: en
+---
+
+## Install the Code Builder Managed Package (Ss much detail is not necessary?)
+
+Salesforce Code Builder is a second-generation (2GP) managed package that you can install in any supported Salesforce org edition. Before you install, make sure you have enabled **Unlocked Packages and Second-Generation Managed Packages** in your Dev Hub.
+
+1. Click the install [link](https://login.salesforce.com/packaging/installPackage.apexp?p0=04t6g000008nwJuAAI)
+2. Log into the org in which you want to install Code Builder.
+3. Check Acknowledge and click **Install**.
+
+
+
+
+![Install Button](../../../images/install_button.png)
+
+4. Grant access and click **Continue**
+
+
+
+![Grant Access](../../../images/grant_access.png)
+
+
+
+5. Click **Done** when installation is Complete.
+
+## Install Code Builder Managed Package (just use this?)
+
+Salesforce Code Builder is a second-generation (2GP) managed package that you can install in any supported Salesforce org edition. Before you install, make sure you have enabled **Unlocked Packages and Second-Generation Managed Packages** in your Dev Hub then follow the standard process for installing a managed package. 
+
+## Assign Permissions
+As an admin, assign permissions to users who will use Code Builder:
+
+1. Click the [link](https://login.salesforce.com/packaging/installPackage.apexp?p0=04t6g000008nwJuAAI) to go to AppExchange and install the Code Builder managed package.
+2. From Setup, enter Permission in the Quick Find box, then select **Permission Sets**.
```

Comment:
```suggestion
2. From Setup, enter `Permission Sets` in the Quick Find box, then select **Permission Sets**.
```
Our style guidelines say to spell it out (for enter) and yes, it feels silly to spell it out twice.